### PR TITLE
Added more tests for tier1, now for mapping

### PR DESCRIPTION
**Pull request by ibragins**

### CODE REVIEW

File: pkg/qe-tests/cypress/integration/tests/tier1/vmware/tier1_tests_mapping.ts [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  });
+
+  beforeEach('Login to MTV', () => {
+    // Restoring local storage and opening base MTV URL
+    cy.restoreLocalStorage();
+    cy.visit(vmwareTier1TestCephCold.loginData.url, { timeout: 120 * SEC });
+    clickByText(button, 'Get started');
+  });
+
+  it('Creating and editing network mapping', () => {
```
Comment (gildub) [inline pkg/qe-tests/cypress/integration/tests/tier1/vmware/tier1_tests_mapping.ts:42]: I think the test description would make sense saying "Creates and edits network mapping".
It a small thing but help when reviewing tests.

File: pkg/qe-tests/cypress/integration/tests/tier1/vmware/tier1_tests_mapping.ts [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    login(vmwareTier1TestCephCold.loginData);
+    getStarted();
+    provider.create(vmwareProviderUser);
+    // Saving local storage state after login
+    cy.saveLocalStorage();
+    createNamespace(vmwareTier1TestCephCold.planData.namespace);
+    provisionNetwork(vmwareTier1TestCephCold.planData.namespace);
+  });
+
+  beforeEach('Login to MTV', () => {
```
Comment (gildub) [inline pkg/qe-tests/cypress/integration/tests/tier1/vmware/tier1_tests_mapping.ts:35]: You might want to change the description on this one to be closer to what the function does.

File: pkg/qe-tests/cypress/integration/tests/tier1/vmware/tier1_tests_mapping.ts [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+} from '../../../../utils/utils';
+import { vmwareTier1TestCephCold } from './tier1_config_vmware';
+import {
+  vmwareNetworkMapping_2x_network,
+  vmwareNetworkMappingSingle,
+  vmwareProviderUser,
+} from './tier1_config_vmware';
+import { button, SEC } from '../../../types/constants';
+import { MappingNetwork } from '../../../models/mappingNetwork';
+// import { MappingStorage } from '../../../models/mappingStorage';
```
Comment (gildub) [inline pkg/qe-tests/cypress/integration/tests/tier1/vmware/tier1_tests_mapping.ts:17]: Remove line if unused

File: pkg/qe-tests/cypress/integration/tests/tier1/vmware/tier1_tests_mapping.ts [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+describe('Tier1 tests for testing provider', () => {
+  const provider = new ProviderVmware();
+  const networkMapping = new MappingNetwork();
+  // const storageMapping = new MappingStorage();
```
Comment (gildub) [inline pkg/qe-tests/cypress/integration/tests/tier1/vmware/tier1_tests_mapping.ts:22]: Remove if unused

File: pkg/qe-tests/cypress/integration/models/mapping.ts [hunk 5]
```diff
@@ -98,7 +111,6 @@ export class Mapping {
     this.selectSourceProvider(sProviderName);
     this.selectTargetProvider(tProviderName);
     this.createMappingPeer(mappingPeer);
-    // cy.wait(2000);
```
Comment (gildub) [inline pkg/qe-tests/cypress/integration/models/mapping.ts:101]: Remove unused line.

**Review by gildub — COMMENTED** [review]
Looks good, just a bit of cleanup and maybe change a couple of descriptions

File: pkg/qe-tests/cypress/integration/tests/tier1/vmware/tier1_tests_mapping.ts (hunk 1 above)
Comment (ibragins) [reply pkg/qe-tests/cypress/integration/tests/tier1/vmware/tier1_tests_mapping.ts:42]: I guess this is exactly the same meaning, this description is shown when test is executed, and it looks better when I see my variant in the test progress

File: pkg/qe-tests/cypress/integration/tests/tier1/vmware/tier1_tests_mapping.ts (hunk 3 above)
Comment (ibragins) [reply pkg/qe-tests/cypress/integration/tests/tier1/vmware/tier1_tests_mapping.ts]: Done

File: pkg/qe-tests/cypress/integration/tests/tier1/vmware/tier1_tests_mapping.ts (hunk 4 above)
Comment (ibragins) [reply pkg/qe-tests/cypress/integration/tests/tier1/vmware/tier1_tests_mapping.ts]: Done

File: pkg/qe-tests/cypress/integration/models/mapping.ts (hunk 5 above)
Comment (ibragins) [reply pkg/qe-tests/cypress/integration/models/mapping.ts:101]: Strange, I don't have it in my code.
Listing of this method is:
```
  protected createDialog(mappingData: MappingData): void {
    const { name, mappingPeer, sProviderName, tProviderName } = mappingData;
    cy.wait(2 * SEC);
    clickByText(button, createMapping);
    this.inputName(name);
    this.selectSourceProvider(sProviderName);
    this.selectTargetProvider(tProviderName);
    this.createMappingPeer(mappingPeer);
    clickByText(buttonModal, create);
    this.validateReady(mappingData);
  }
  ```

Comment (gildub) [reply pkg/qe-tests/cypress/integration/models/mapping.ts:101]: You're right ! Sorry this line was removed. Never mind.

**Pull request merged**